cmd/algorand-indexer/daemon.go

```diff
@@ -103,7 +109,6 @@ var daemonCmd = &cobra.Command{
 
 				logger.Info("Starting block importer.")
 				bot.Run()
-				cf()
```

Comment:
There's another one on line 64 that can be removed.

Reply:
We need that one to cancel the context when Indexer is politely killed.

Reply:
Gotcha, calling cf there causes `bot.Run` and `api.Serve` to terminate on an exit signal.